README.md

```diff
@@ -5,40 +5,37 @@
-- **Owner**: @davidraleigh
+- **Owner**: @davidraleigh @matthewhanson
 
 This document explains the Storage Extension to the [SpatioTemporal Asset Catalog](https://github.com/radiantearth/stac-spec) (STAC) specification.
 This extension adds fields to STAC Item and Asset objects, allowing for details related to cloud storage access and costs to be associated
 with a STAC Item.  This extension does not cover NFS solutions provided by PaaS cloud companies.
 
 - Examples:
-  - [Item example 1](examples/example-naip.json): Shows the basic usage of the extension in a STAC Item.
-  - [Item example 2](examples/example-nsl.json): Another example of basic usage.
+  - [Item example 1](examples/item-naip.json): Shows the basic usage of the extension in a STAC Item.
+  - [Item example 2](examples/item-nsl.json): Another example of basic usage.
 - [JSON Schema](json-schema/schema.json)
 - [Changelog](./CHANGELOG.md)
 
 ## Item Fields
+
 | Field Name  | Type   | Description |
 | ----------- | ------ | ----------- |
 | storage:min_tier_duration   | integer  | number of days for the shortest time tier restriction on access of an asset. |
 | storage:max_tier_duration   | integer  | number of days for the longest time tier restrictions on access of an asset. |
 | storage:archived            | bool    | descriptor for whether the data is "properly" archived according to implementers discretion |
-
-## Asset Fields
-
-| Field Name  | Type   | Description |
-| ----------- | ------ | ----------- |
 | storage:platform              | string    | (REQUIRED) The [cloud provider](#providers) where data is stored |
-| storage:manager               | string    | The entity in charge of managing the data. |
 | storage:region                | string    | (REQUIRED) The region where the data is stored. Relevant to speed of access and inter region egress costs (as defined by PaaS provider) |
-| storage:bucket                | string    | The bucket for the asset, used along with object path |
+| storage:bucket                | string    | The bucket for the asset(s), used along with object path |
 | storage:object_path           | string    | The object_path for the asset, used along with bucket |
-| storage:requester_pays        | bool      | (REQUIRED) Is the data requester pays or is it data manager/cloud provider pays |
+| storage:requester_pays        | bool      | Is the data requester pays or is it data manager/cloud provider pays. *Defaults to false* |
 | storage:tier                  | string    | The title for the tier type (as defined by PaaS provider) |
 | storage:tier_duration         | integer   | Minimum storage duration (in days) required before additional fees |
-| storage:date_stored           | string    | Date and time the corresponding asset placed into the current storage tier (relevant for tier_duration > 0). Format is RFC 3339. |
```

Comment:
This is useful for keeping track of budgeting and how often a data set is pulled out of a "coldish" storage before it's "ready". but if you want to drop it, maybe it's too niche.

Reply:
It seems pretty specific, but also incomplete if you really wanted to do through tracking of data management. Seems like that should be tracked with an internal extension or as part of the system that manages storage locations.